src/components/Backdrop/tests/Backdrop.test.tsx

```diff
@@ -1,24 +1,25 @@
 import React from 'react';
-// eslint-disable-next-line no-restricted-imports
-import {mountWithAppProvider, findByTestID} from 'test-utilities/legacy';
+import {mountWithApp} from 'test-utilities';
 
 import {Backdrop} from '..';
 
 describe('<Backdrop />', () => {
   describe('onDismiss()', () => {
     it('is called when the backdrop is clicked', () => {
       const spy = jest.fn();
-      const backdrop = mountWithAppProvider(<Backdrop onClick={spy} />);
-      findByTestID(backdrop, 'Backdrop').simulate('click');
+      const backdrop = mountWithApp(<Backdrop onClick={spy} />);
+      backdrop.find('div')!.trigger('onClick');
```

Comment:
Simple enough component that I don't think it's a real issue but ideally we wouldn't target elements this vaguely since it can be quite brittle. If another `div` were added to this component it would start failing and you've have to reverse engineer which random div it was triggering the action on. Again, not _reaaally_ an issue for this particular component because of how small it is but I'd love for us to think about how we can target these in the future.

Reply:
Targeted the <div> using the `onClick` prop.